unlock-app/src/config/queryClient.ts

```diff
@@ -6,37 +6,32 @@ import * as Sentry from '@sentry/nextjs'
 export const queryClient = new QueryClient({
   queryCache: new QueryCache({
     onError: (error: any, query) => {
+      if (error instanceof AxiosError) {
```

Comment:
I think we should keep sending these to Sentry... please move after the sentry reporting